lbrynet/conf.py

```diff
@@ -283,7 +283,7 @@ def _convert_value(value):
     # if reflect_uploads is True, send files to reflector (after publishing as well as a
     # periodic check in the event the initial upload failed or was disconnected part way through
```

Comment:
maybe update this comment? `reflect_uploads` doesn't control the re-reflect anymore

Reply:
Sorry, it also does, but this became a bit confusing. Maybe the `# set to 0 to disable` added below is enough. Or explain that they both need to be active for enabling re-reflect.

Reply:
If `reflect_uploads` is true, streams made with `publish` will be uploaded to reflector. If `reflect_uploads` is true and `auto_re_reflect_interval` > 0 files in the file manager will be reflected.